colossalai/accelerator/cuda_accelerator.py

```diff
@@ -54,3 +64,227 @@ def device_count(self):
         Return the number of devices on the machine.
         """
         return torch.cuda.device_count()
+
+    def set_to_device(self, models: Any) -> Any:
+        """
+        Send model to gpu.
+
+        :param models: nn.module or a list of module
+        """
+        if isinstance(models, list) and len(models) > 1:
+            ret = []
+            for model in models:
+                ret.append(model.to(self.get_current_device()))
```

Comment:
It seems this function is similar to that of cpu accelerator. Can we define it in base class?

Reply:
ok